landoapi/uplift.py

```diff
@@ -197,3 +208,12 @@ def create_approval_request(phab: PhabricatorClient, revision: dict, form_conten
         "revision_id": rev_id,
         "revision_phid": rev_phid,
     }
+
+
+def stack_uplift_form_submitted(stack_data) -> bool:
+    """Return `True` if the stack has a valid uplift request form submitted."""
+    # TODO just checks that any of the revisions in the stack have the uplift form
+    # submitted.
```

Comment:
This check is basically "does some revision in the stack have an uplift request form submitted". The `TODO` is mostly to check if others have opinions on whether this should be a more complicated check. For example we could check the tip commit of the changeset being landed, which would mean partial landings of stacks require a submitted form on each submitssion.

Reply:
Maybe this should be a `NOTE`?
```suggestion
    # NOTE: this just checks that any of the revisions in the stack have the uplift
    # form submitted.
```